src/services/role/format.ts

```diff
@@ -0,0 +1,39 @@
+import cuid from 'cuid'
+import { V1Role } from '@kubernetes/client-node'
+import { K8sRole } from '../../types/generated'
+import formatMetadata from '../../util/metadata'
+
+export default ({
+  entity,
+  context,
+}: {
+  entity: V1Role
+  context: string
+}): K8sRole => {
+  const {
+    apiVersion,
+    kind,
+    metadata,
+    rules
+  } = entity
+
+  const formattedMetadata = formatMetadata(metadata)
+
+  const mappedRules = rules?.map(({ apiGroups, nonResourceURLs, resources, resourceNames, verbs }) => ({
+    id: cuid(),
+    apiGroups,
+    nonResourceUrls: nonResourceURLs,
+    resources,
+    resourceNames,
+    verbs
+  })) ?? []
+
+  return {
+    id: formattedMetadata.id,
+    apiVersion,
```

Comment:
I dont this is related to this pr specifically but it looks lie apiVersion is null across all services

Reply:
yea I dont know why none of the services in our sandbox have them filled in but they are a part of the obj definition. I can ask Chris K.